device/mellanox/x86_64-nvidia_sn5610n-r0/Mellanox-SN5610N-C256S2/qos.json.j2

```diff
@@ -0,0 +1,222 @@
+{#
```

Comment:
Can you please recheck this file? Its contents are having buffer macros while it should have macros for qos tables